projects/ngneat/transloco/src/lib/transloco.directive.ts

```diff
@@ -70,6 +72,14 @@ export class TranslocoDirective implements OnInit, OnDestroy, OnChanges {
           const lang = this.getLang();
           const scope = this.getScope();
           this.langName = scope ? `${scope}/${lang}` : lang;
+          if (
+            !this.inlineScope &&
+            this.providerScope &&
+            isTranslocoScopeInterface(this.providerScope) &&
+            this.configProvider
```

Comment:
There is no need to inject the config. Create a method in the service and use it.